packages/venia-ui/package.json

```diff
@@ -54,7 +55,7 @@
     "@babel/plugin-transform-runtime": "~7.4.4",
     "@babel/preset-env": "~7.3.4",
     "@storybook/react": "~4.1.13",
-    "apollo-client": "2.4.13",
+    "apollo-client": "2.6.4",
```

Comment:
Likewise, make sure this same version is in Peregrine's `package.json` as a peer dependency.